RecoTauTag/RecoTau/test/rerunTauRecoOnMiniAOD.py

```diff
@@ -0,0 +1,119 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Hi @slava77 , 
I'm currently looking into this and would add a runtest as I know them from he past and as done e.g. here: https://github.com/cms-tau-pog/cmssw/blob/CMSSW_10_0_X_tau-pog/TopQuarkAnalysis/TopEventProducers/test/runtestTqafTopEventProducers.cpp. Is this what you had in mind? Otherwise can you point me to the test you are referring to? 

Thanx,
Roger